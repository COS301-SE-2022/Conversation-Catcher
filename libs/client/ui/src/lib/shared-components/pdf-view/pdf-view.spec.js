import React from 'react';
import { render } from '@testing-library/react-native';
import PdfView from './pdf-view';
import { create } from 'react-test-renderer';
describe('PdfView', () => {
  it('should render successfully', () => {
    const { container } = render(<PdfView />);
    expect(container).toBeTruthy();
  });

  
  const tree = create(<PdfView />);
  test('snapshot', () => {
    expect(tree).toMatchSnapshot();
  });
  
});

// describe('The actions that occur on button press', () => {
//   it('should display the available colours')
// });
