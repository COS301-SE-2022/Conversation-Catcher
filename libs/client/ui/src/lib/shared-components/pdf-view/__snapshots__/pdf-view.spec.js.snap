// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PdfView snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffffff",
      "flexGrow": 1,
      "marginBottom": 0,
      "marginLeft": 0,
      "marginRight": 0,
      "marginTop": 0,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#c4c4c4ff",
        "elevation": 2,
        "flexDirection": "row",
        "flexShrink": 1,
        "shadowColor": "#000000",
        "shadowOffset": Object {
          "height": 1,
          "width": 0,
        },
        "shadowOpacity": 0.2173913043478261,
        "shadowRadius": 2.621621621621622,
        "top": 0,
        "width": "100%",
        "zIndex": 999,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "flex-start",
          "flexGrow": 1,
          "justifyContent": "center",
          "minHeight": 28,
          "padding": 15,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#344053ff",
            "fontFamily": "System",
            "fontSize": 24,
            "fontStyle": "normal",
            "fontWeight": "700",
            "letterSpacing": 0,
            "lineHeight": 28,
            "textAlign": "center",
          }
        }
      />
    </View>
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexGrow": 1,
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <Text
        allowFontScaling={false}
        selectable={false}
        style={
          Array [
            Object {
              "color": "#344053ff",
              "fontSize": 30,
            },
            undefined,
            Object {
              "fontFamily": "FontAwesome",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "height": "70%",
        "overflow": "scroll",
        "padding": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#344053ff",
          "fontFamily": "System",
          "fontSize": 16,
          "fontStyle": "normal",
          "fontWeight": "400",
          "letterSpacing": 0,
          "lineHeight": 20,
          "paddingHorizontal": 0,
          "textAlign": "left",
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignSelf": "flex-end",
        "backgroundColor": "#c4c4c4ff",
        "borderColor": "#d0d5ddff",
        "borderRadius": 5,
        "borderStyle": "solid",
        "borderWidth": 1,
        "bottom": 0,
        "elevation": 2,
        "flexDirection": "row",
        "flexShrink": 1,
        "justifyContent": "center",
        "position": "absolute",
        "shadowColor": "#000000",
        "shadowOffset": Object {
          "height": 1,
          "width": 0,
        },
        "shadowOpacity": 0.2173913043478261,
        "shadowRadius": 2.621621621621622,
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "flexGrow": 1,
          "justifyContent": "center",
          "opacity": 1,
          "padding": 10,
        }
      }
    >
      <Text
        allowFontScaling={false}
        selectable={false}
        style={
          Array [
            Object {
              "color": "#344053ff",
              "fontSize": 30,
            },
            undefined,
            Object {
              "fontFamily": "FontAwesome",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
  </View>
  <Modal
    animationType="none"
    deviceHeight={null}
    deviceWidth={null}
    hardwareAccelerated={false}
    hideModalContentWhileAnimating={false}
    onBackdropPress={[Function]}
    onModalHide={[Function]}
    onModalWillHide={[Function]}
    onModalWillShow={[Function]}
    onRequestClose={[Function]}
    panResponderThreshold={4}
    scrollHorizontal={false}
    scrollOffset={0}
    scrollOffsetMax={0}
    scrollTo={null}
    statusBarTranslucent={false}
    supportedOrientations={
      Array [
        "portrait",
        "landscape",
      ]
    }
    swipeThreshold={100}
    transparent={true}
    visible={false}
  />
</View>
`;
